src/components/TextEditor.vue

```diff
@@ -292,10 +293,7 @@ export default {
 			try {
 				logger.debug(`loading ${language} translations for CKEditor`)
 				await import(
-					/* webpackMode: "lazy-once" */
-					/* webpackPrefetch: true */
-					/* webpackPreload: true */
-					`@ckeditor/ckeditor5-build-decoupled-document/build/translations/${language}`
+					`../../node_modules/@ckeditor/ckeditor5-build-decoupled-document/build/translations/${language}.js`
```

Comment:
Revert?

Or was this changed on purpose?

Reply:
On purpose, for some reason vite is unable to find  `@ckeditor/`

Reply:
this should be a separate PR at least. If composer translations are broken we have to backport the fix

Reply:
I'll split it